src/main/java/frc/robot/commands/indexer/IndexerCommand.java

```diff
@@ -4,6 +4,8 @@
 
 package frc.robot.commands.indexer;
 
+import static frc.robot.Constants.IntakeConstants.*;
```

Comment:
Shouldn't this be using IndexerConstants?

Reply:
Uh yes, one sec